Mathlib/Data/QPF/Multivariate/Basic.lean

```diff
@@ -130,7 +130,7 @@ theorem liftP_iff {α : TypeVec n} (p : ∀ ⦃i⦄, α i → Prop) (x : F α) :
   use abs ⟨a, fun i j => ⟨f i j, h₁ i j⟩⟩
   rw [← abs_map, h₀]; rfl
 
-theorem liftR_iff {α : TypeVec n} (r : ∀ /- ⦃i⦄ -/ {i}, α i → α i → Prop) (x y : F α) :
+theorem liftR_iff {α : TypeVec n} (r : ∀ {i}, α i → α i → Prop) (x y : F α) :
```

Comment:
```suggestion
theorem liftR_iff {α : TypeVec n} (r : ∀ ⦃i⦄, α i → α i → Prop) (x y : F α) :
```
It looks to me like this might have been the intention? eg in `liftP_iff`, this binder is used instead.

Reply:
Sure, that's also possible. Do you think an instance-implicit binder is better, then I'll adopt your suggestion instead.

Reply:
I think the comment here predates the existence of the `⦃i⦄` binders in Lean 4.

Reply:
Thanks for looking this up - I've made it an instance-implicit binder instead.